website/www/site/content/en/documentation/programming-guide.md

```diff
@@ -1184,6 +1184,91 @@ func init() {
 
 </span>
 
+{{< paragraph class="language-python">}}
+Proper Use of return vs yield in Python Functions.
+{{< /paragraph >}}
+
+{{< highlight python >}}
+# Returning a single string instead of a sequence
```

Comment:
Rather than having a list of examples at the top, could we interleave these examples with the descriptions below?